packages/ODEs/src/css/app-styles.css

```diff
@@ -0,0 +1,13 @@
+.demo-app-div-help {
```

Comment:
What is ".demo-app-div-help"? The package should not modify global styles. And if it's local, it should be prefixed with "eqs" or whatever the package prefix is

Reply:
Fixed